ts/src/test/test.ts

```diff
@@ -205,8 +206,32 @@ export default class testMainClass extends baseMainTestClass {
         };
         const exchange = initExchange (exchangeId, exchangeArgs);
         await this.importFiles (exchange);
-        this.expandSettings (exchange, symbol);
-        await this.startTest (exchange, symbol);
+        this.expandSettings (exchange);
+        symbolArgv = this.checkIfSpecificTestIsChosen (symbolArgv);
```

Comment:
Hum when I try to select a test I get this error

```
 node js/src/test/test.js bitget fetchTicker 
Debugger attached.

TESTING  js { exchange: 'bitget', symbol: 'fetchTicker' } 

Exchange loaded 894 symbols BTC/EUR, BTC/USDT, DOGE/BTC, ETH/BTC, ETH/EUR + more...
file:///Users/cjg/Git/ccxt10/ccxt/js/src/test/test.js:11
    throw new Error(exceptionMessage(e));
    ^

Error: [Error] Error: [BadSymbol] BadSymbol: bitget does not have market symbol fetchTicker
    at bitget.market (file:///Users/cjg/Git/ccxt10/ccxt/js/src/base/Exchange.js:3543:15)
    at testMainClass.testExchange (file:///Users/cjg/Git/ccxt10/ccxt/js/src/test/test.js:638:37)
    at testMainClass.startTest (file:///Users/cjg/Git/ccxt10/ccxt/js/src/test/test.js:801:24)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async testMainClass.init (file:///Users/cjg/Git/ccxt10/ccxt/js/src/test/test.js:185:9)
    at process.<anonymous> (file:///Users/cjg/Git/ccxt10/ccxt/js/src/test/test.js:15:11)
    at process.emit (node:events:517:28)
    at emit (node:internal/process/promises:149:20)
    at processPromiseRejections (node:internal/process/promises:283:27)
    at process.processTicksAndRejections (node:internal/process/task_queues:96:32)
    at process.<anonymous> (file:///Users/cjg/Git/ccxt10/ccxt/js/src/test/test.js:11:11)
```

Reply:
@carlosmiei I've fixed. if you can plz take a look into this PR, as I have to merge this soon into other tests-related PRs

Reply:
@ttodua I get the same eror in python 

```
 python python/ccxt/test/test_async.py bybit fetchTicker --sandbox

TESTING  py {'exchange': 'bybit', 'symbol': 'fetchTicker'} 

Exchange loaded 1135 symbols BTC/EUR, BTC/USDT, ETH/BTC + more...
<class 'ccxt.base.errors.BadSymbol'> bybit does not have market symbol fetchTicker 
<UNHANDLED EXCEPTION>
  File "/Users/cjg/Git/ccxt10/ccxt/python/ccxt/test/test_async.py", line 1263, in <module>
    asyncio.run(testMainClass().init(argv.exchange, argv.symbol))

  File "/opt/homebrew/Cellar/python@3.11/3.11.5/Frameworks/Python.framework/Versions/3.11/lib/python3.11/asyncio/runners.py", line 190, in run
    return runner.run(main)
           ^^^^^^^^^^^^^^^^

  File "/opt/homebrew/Cellar/python@3.11/3.11.5/Frameworks/Python.framework/Versions/3.11/lib/python3.11/asyncio/runners.py", line 118, in run
    return self._loop.run_until_complete(task)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Reply:
hm, did you run transpile at first? test.py/php are not pushed here by transpiled version

Reply:
I transpiled everything twice just to make sure

Reply:
ready

Reply:
@ttodua  will test again